### PR TITLE
code cleanup

### PR DESCRIPTION
**Pull request by yangfl**

### CODE REVIEW

**Comment by majestrate** [comment]
Since this is a huge diff I'd like to get approval from others on this before merging it.

File: libi2pd/Destination.h [hunk 1]
```diff
@@ -239,7 +239,7 @@ namespace client
 			int m_StreamingAckDelay;
 			std::shared_ptr<i2p::stream::StreamingDestination> m_StreamingDestination; // default
 			std::map<uint16_t, std::shared_ptr<i2p::stream::StreamingDestination> > m_StreamingDestinationsByPorts;
-      		i2p::datagram::DatagramDestination * m_DatagramDestination;
+		i2p::datagram::DatagramDestination * m_DatagramDestination;
```
Comment (r4sas) [inline libi2pd/Destination.h]: replace to <tab>

File: libi2pd/TransitTunnel.h [hunk 2]
```diff
@@ -14,34 +14,34 @@
...
+	class TransitTunnel: public TunnelBase
 	{
 		public:
 
 			TransitTunnel (uint32_t receiveTunnelID,
-			    const uint8_t * nextIdent, uint32_t nextTunnelID, 
-	    		const uint8_t * layerKey,const uint8_t * ivKey); 
-			
+			    const uint8_t * nextIdent, uint32_t nextTunnelID,
+			const uint8_t * layerKey,const uint8_t * ivKey);
```
Comment (r4sas) [inline libi2pd/TransitTunnel.h]: add tab

File: libi2pd/TransitTunnel.h [hunk 3]
```diff
@@ -14,34 +14,34 @@
...
 	class TransitTunnelParticipant: public TransitTunnel
 	{
 		public:
 
 			TransitTunnelParticipant (uint32_t receiveTunnelID,
-			    const uint8_t * nextIdent, uint32_t nextTunnelID, 
-	    		const uint8_t * layerKey,const uint8_t * ivKey):
-				TransitTunnel (receiveTunnelID, nextIdent, nextTunnelID, 
+			    const uint8_t * nextIdent, uint32_t nextTunnelID,
+			const uint8_t * layerKey,const uint8_t * ivKey):
```
Comment (r4sas) [inline libi2pd/TransitTunnel.h]: add tab

File: libi2pd/TransitTunnel.h [hunk 4]
```diff
@@ -53,51 +53,51 @@ namespace tunnel
...
 	class TransitTunnelGateway: public TransitTunnel
 	{
 		public:
 
 			TransitTunnelGateway (uint32_t receiveTunnelID,
-			    const uint8_t * nextIdent, uint32_t nextTunnelID, 
-	    		const uint8_t * layerKey,const uint8_t * ivKey):
-				TransitTunnel (receiveTunnelID, nextIdent, nextTunnelID, 
+			    const uint8_t * nextIdent, uint32_t nextTunnelID,
+			const uint8_t * layerKey,const uint8_t * ivKey):
```
Comment (r4sas) [inline libi2pd/TransitTunnel.h]: add tab

File: libi2pd/TransitTunnel.h [hunk 5]
```diff
@@ -53,51 +53,51 @@ namespace tunnel
...
 
 	class TransitTunnelEndpoint: public TransitTunnel
 	{
 		public:
 
 			TransitTunnelEndpoint (uint32_t receiveTunnelID,
-			    const uint8_t * nextIdent, uint32_t nextTunnelID, 
-	    		const uint8_t * layerKey,const uint8_t * ivKey):
+			    const uint8_t * nextIdent, uint32_t nextTunnelID,
+			const uint8_t * layerKey,const uint8_t * ivKey):
```
Comment (r4sas) [inline libi2pd/TransitTunnel.h]: add tab

**Comment by yangfl** [comment]
ok

**Comment by r4sas** [comment]
Thanks!

**Pull request merged**